colibre/auto_plotter/cold_gas_relations.yml

```diff
@@ -350,4 +354,534 @@ H2_mass_star_formation_rate_100:
   metadata:
     title: H2 Gas Mass - Star Formation Rate
     caption: All galaxies are included in the median line.
-    section: Cold Gas Relations
+    section: Original Cold Gas Relations
+
+stellar_mass_neutral_H_to_baryonic_fraction_100:
+  type: "scatter"
+  legend_loc: "upper right"
+  x:
+    quantity: "apertures.mass_star_100_kpc"
+    units: Solar_Mass
+    start: 1e5
+    end: 1e12
+  y:
+    quantity: "derived_quantities.gas_neutral_H_to_baryonic_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+    lower: 
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Stellar Mass-Neutral Gas to Baryonic Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is HI + H_2 mass over total baryonic mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs M*
+
+stellar_mass_HI_to_neutral_H_fraction_100:
+  type: "scatter"
+  legend_loc: "upper right"
+  x:
+    quantity: "apertures.mass_star_100_kpc"
+    units: Solar_Mass
+    start: 1e5
+    end: 1e12
+  y:
+    quantity: "derived_quantities.gas_HI_to_neutral_H_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-1
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Stellar Mass-HI to Neutral Gas Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is HI mass over HI + H2 mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs M*
+
+stellar_mass_H2_to_neutral_H_fraction_100:
+  type: "scatter"
+  legend_loc: "upper right"
+  x:
+    quantity: "apertures.mass_star_100_kpc"
+    units: Solar_Mass
+    start: 1e5
+    end: 1e12
+  y:
+    quantity: "derived_quantities.gas_H2_to_neutral_H_fraction_100_kpc"
+    units: "dimensionless"
+    start: 1e-4
+    end: 1.1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 15
+    start:
+      value: 1e5
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+    lower:
+      value: 0
+      units: "dimensionless"
+    upper:
+      value: 1e99
+      units: "dimensionless"
+  metadata:
+    title: Stellar Mass-H2 to Neutral Gas Fraction (100 kpc Stellar Mass)
+    caption: All galaxies are included in the median line. Fraction is H2 mass over HI + H2 mass in a 100 kpc aperture.
+    section: Cold Gas Fractions vs M*
```

Comment:
Text should be well-formatted everywhere; can we do `Cold Gas Fractions (Stellar Mass)` or `Cold Gas Fractions ($M_*$)`?